test/model.test.js

```diff
@@ -7302,6 +7302,19 @@ describe('Model', function() {
       /First argument to `Model` constructor must be an object/
     );
   });
+
```

Comment:
Suggestion: would add another test case verifying that the schema isn't modified without calling `recompileSchema`. Unless that's implicitly tested somewhere else.

Reply:
There's a part in the "supports recompiling model with new schema additions" test case that asserts `myVirtual()` isn't applied before `recompileSchema()` is called.